### PR TITLE
Update Introduction to Generic Pipelines tutorial

### PR DESCRIPTION
**Pull request by sanika93**
Proposed Changes -

1. Added a note in the Setup section of the generic pipelines tutorial which will help new developers understand how to launch JupyterLab from the correct directory. 
2. While following the introduction to generic pipelines tutorial, I found that the following images need to be updated:  
     i. In Section - `Add a notebook or script to the pipeline` steps 1 and 2, the component palette panel on the left hand side displays `Elyra` in the header instead of `Generic`
    ii. In Section - `Add a notebook or script to the pipeline`, step 9, the UI displays a new environment variable `ELYRA_RUNTIME_ENV`, hence updated the image. I read in this doc - https://elyra.readthedocs.io/en/latest/user_guide/best-practices-file-based-nodes.html#elyra-runtime-env that it can be set to local. I tested out the example by setting it to local and it worked. However even leaving that variable empty, the tutorial worked. Hence added a new line explaining the variable and that it can be left optional.

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: pipelines/introduction-to-generic-pipelines/README.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ In this introductory tutorial you will learn how to create a generic pipeline an
 ### Setup
 
 This tutorial uses the `introduction to generic pipelines` sample from the https://github.com/elyra-ai/examples GitHub repository.
-1. Launch JupyterLab.
+1. Launch JupyterLab.    
+   > **Note**: When you start JupyterLab using the `jupyter lab` command, it loads the contents of the current working directory.  
+   > For examples, we recommend creating a new directory on your system and running the `jupyter lab` command from that directory.
```
Comment (karlaspuldaro) [inline pipelines/introduction-to-generic-pipelines/README.md:47]: I believe the motivation behind this extra info is to also to make sure the root directory used to run juypterlab is not an existing git repository, otherwise the next step below won't work. Maybe it's worth mentioning that somehow here.

Comment (ptitzler) [reply pipelines/introduction-to-generic-pipelines/README.md:47]: Yes!

File: pipelines/introduction-to-generic-pipelines/README.md [hunk 2]
```diff
@@ -139,7 +141,7 @@ To add a notebook or script to the pipeline:
 
    ![Scan file for environment variables](doc/images/scan-file.png)
 
-   It appears that `load_data` requires environment variable `DATASET_URL` to be set. This variable identifies the name and location of a data set file, which the notebook or script will download and decompress. 
+   It appears that `load_data` references two environment variables i.e `DATASET_URL` and `ELYRA_RUNTIME_ENV`. The `DATASET_URL` requires to be set. This variable identifies the name and location of a data set file, which the notebook or script will download and decompress. The `ELYRA_RUNTIME_ENV` is optional. It can be set to `local`.
```
Comment (ptitzler) [inline pipelines/introduction-to-generic-pipelines/README.md:144]: > The `ELYRA_RUNTIME_ENV` is optional. It can be set to `local`.  

TL;DR. This is a read-only variable. For details refer to https://elyra.readthedocs.io/en/stable/user_guide/best-practices-file-based-nodes.html#proprietary-environment-variables.

File: pipelines/introduction-to-generic-pipelines/README.md [hunk 3]
```diff
@@ -42,7 +42,9 @@ In this introductory tutorial you will learn how to create a generic pipeline an
 ### Setup
 
 This tutorial uses the `introduction to generic pipelines` sample from the https://github.com/elyra-ai/examples GitHub repository.
-1. Launch JupyterLab.
+1. Launch JupyterLab.    
+   > **Note**: When you start JupyterLab using the `jupyter lab` command, it loads the contents of the current working directory.  
+   > For examples, we recommend starting JupyterLab from a new directory on your system that is not an existing git repository. This will make sure you can clone the repository as mentioned in the next step.
```
Comment (ptitzler) [inline pipelines/introduction-to-generic-pipelines/README.md:47]: Typo:

```suggestion
   > For example, we recommend starting JupyterLab from a new directory on your system that is not an existing git repository. This will make sure you can clone the repository as mentioned in the next step.
```

Comment (sanika93) [reply pipelines/introduction-to-generic-pipelines/README.md:47]: Do you think it is a good idea to replace it with - For `tutorials` instead of For `examples` ? The context here is to create a new directory when the user wants to try out the Elyra tutorials.

Comment (ptitzler) [reply pipelines/introduction-to-generic-pipelines/README.md:47]: We should not, because it is likely that many users will use github to share/backup their local JupyterLab artifacts (notebooks etc), not just those going through the tutorial.

Comment (sanika93) [reply pipelines/introduction-to-generic-pipelines/README.md:47]: okay sounds good! Thanks 👍

**Review by ptitzler — APPROVED** [review]
LGTM! Thank you!
